rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/test_is_hvac_sys_preheating_type_elec_resistance.py

```diff
@@ -0,0 +1,65 @@
+from rct229.ruleset_functions.baseline_systems.baseline_hvac_sub_functions.is_hvac_sys_preheating_type_elec_resistance import (
+    is_hvac_sys_preheating_type_elec_resistance,
+)
+
+TEST_RMD = {
+    "id": "test_rmd",
+    "buildings": [
+        {
+            "id": "building_1",
+            "building_segments": [
+                {
+                    "id": "building_segment_1",
+                    "heating_ventilation_air_conditioning_systems": [
+                        {
+                            "id": "hvac_1",
+                            "preheat_system": {
+                                "id": "preheat_system",
+                                "hot_water_loop": "HW_Loop_1",
+                                "heating_system_type": "FLUID_LOOP",
+                            },
+                        },
+                        {
+                            # Case where the preheat system has a wrong hot_water_loop id
+                            "id": "hvac_2",
+                            "preheat_system": {
+                                "id": "preheat_system",
+                                "heating_system_type": "ELECTRIC_RESISTANCE",
+                            },
+                        },
+                        {
+                            # Case where there is no preheat system
+                            "id": "hvac_3",
+                        },
+                    ],
+                }
+            ],
+        }
+    ],
+    "boilers": [
+        {
+            "id": "boiler_1",
+            "loop": "HW_Loop_1",
+        }
+    ],
+    "fluid_loops": [
+        {
+            "id": "HW_Loop_1",
+            "type": "HEATING",
+        }
+    ],
+}
+
+TEST_RMD_FULL = {"id": "229_01", "ruleset_model_instances": [TEST_RMD]}
+
+
+def test__preheatheating_type_fluid_loop():
```

Comment:
Shouldn't the test name match the function being tested?
Also, there should be an extra underscore between "..._type" and "fluid_loop".